### DISCUSSION TITLE
11.0.0 Release Candidate 2.1

### DISCUSSION THREAD

**Discussion by danwalmsley** [AvaloniaUI/Avalonia#11983] [General]
List of breaking changes (hopefully the last ones before stable release) in rc2:
https://github.com/AvaloniaUI/Avalonia/pulls?q=is%3Apr+is%3Amerged+label%3Abreaking-change+closed%3A2023-06-01..2023-07-01+

Other big changes in rc2:
- [macOS experimental Metal support](https://github.com/AvaloniaUI/Avalonia/pull/11914)
- [android single activity support](https://github.com/AvaloniaUI/Avalonia/pull/11854)

<hr /><em>This discussion was created from the release <a href='https://github.com/AvaloniaUI/Avalonia/releases/tag/11.0.0-rc2.1'>11.0.0 Release Candidate 2.1</a>.</em>

**Comment by Numpsy** [comment]
I've tried updating https://github.com/fsprojects/Avalonia.FuncUI to RC 2.1 and it all seems to work ok when building from the command line or in VSCode, but fwiw I get an error like the one shown at https://github.com/AvaloniaUI/Avalonia/issues/11913 if I try to build in Visual Studio 2022.

**Reply by danwalmsley** [reply]
https://github.com/AvaloniaUI/Avalonia/pull/11985

fix coming.

**Comment by danwalmsley** [comment]
https://www.nuget.org/packages/Avalonia/11.0.0-rc2.2
